test/c3/appframework/appframework_test.go

```diff
@@ -127,11 +127,27 @@ var _ = Describe("c3appfw test", func() {
 			// Wait for the poll period for the apps to be downloaded
 			time.Sleep(2 * time.Minute)
 
+			// Ensure that the cluster-master goes to Ready phase
+			testenv.ClusterMasterReady(deployment, testenvInstance)
+
+			// Ensure indexers go to Ready phase
+			testenv.SingleSiteIndexersReady(deployment, testenvInstance)
+
+			// Ensure search head cluster go to Ready phase
+			testenv.SearchHeadClusterReady(deployment, testenvInstance)
+
+			// Verify MC Pod is Ready
+			testenv.MCPodReady(testenvInstance.GetName(), deployment)
+
+			// Verify RF SF is met
+			testenv.VerifyRFSFMet(deployment, testenvInstance)
+
 			// Verify Apps are downloaded by init-container
 			testenv.VerifyAppsDownloadedByInitContainer(deployment, testenvInstance, testenvInstance.GetName(), podNames, appListV2, initContDownloadLocation)
 
 			//Verify Apps are copied to location
-			testenv.VerifyAppsCopied(deployment, testenvInstance, testenvInstance.GetName(), allPodNames, appListV2, true, true)
+			appListV1 = append(appListV1, testenv.PollAfterApps...)
```

Comment:
As per our discussion, the reason for continuing to use `appListV1` rather than `appListV2` is because of the way the map `AppInfo` has been structured - It would be better for the map to have the key as `App-name` rather than how i tis now (where the V1-file name is the key

This will bring in some changes to the tests, Twain is exploring further. cc: @jambrosiano , @pdhanoyasplunk